en/docs/updates/registry-image-verification.md

```
@@ -2,19 +2,19 @@
 Before verification, ensure:
 
-* [Cosign](https://docs.sigstore.dev/cosign/system_config/installation/) is installed  
-* Access to the [WSO2 container registry](https://registry.wso2.com)  
+* [Cosign](https://docs.sigstore.dev/cosign/system_config/installation/) is installed
+* Access to the [WSO2 container registry](https://registry.wso2.com)
 * Public key provided by WSO2
 
 You need the WSO2 public key to validate the signatures. Save the following block into a file named **wso2-public-key.pub** on your local machine:
 
-```shell
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)